specification/oep/resource-manager/Microsoft.OpenEnergyPlatform/preview/2021-06-01-preview/oep.json

```
@@ -0,0 +1,486 @@
+          }
+        },
+        "x-ms-examples": {
+          "OepResource_Delete": {
+            "$ref": "./examples/OepResource_Delete.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.OpenEnergyPlatform/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "description": "Lists the available operations of Microsoft.OpenEnergyPlatform resource provider.",
+        "operationId": "Operations_List",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        },
+        "x-ms-examples": {
+          "Operations_List": {
+            "$ref": "./examples/Operations_List.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "EnergyServicesList": {
+      "description": "The list of oep resources.",
+      "type": "object",
+      "properties": {
+        "nextLink": {
+          "description": "The link used to get the next page of oep resources list.",
+          "type": "string"
+        },
+        "value": {
+          "description": "The list of oep resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EnergyServices"
+          }
+        }
+      }
+    },
+    "Item": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        }
+      }
+    },
+    "OepProperties": {
+      "type": "object",
+      "properties": {
+        "dnsName": {
+          "type": "string",
+          "readOnly": true
+        },
+        "provisioningState": {
+          "enum": [
+            "Unknown",
+            "Succeeded",
+            "Failed",
+            "Canceled",
+            "Creating",
+            "Deleting",
+            "Updating"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "authAppId": {
+          "type": "string",
+          "readOnly": false
+        },
+        "dataPartitionNames": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Item"
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "EnergyServices": {
+      "required": [
+        "location"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/OepProperties"
```

Could we also rename this model to `EnergyServiceProperties` to reduce the possible confusion here?
```suggestion
          "$ref": "#/definitions/EnergyServiceProperties"
```